src/rajAJun20/AllDifferIndexString.java

```diff
@@ -0,0 +1,29 @@
+package rajAJun20;
+
+public class AllDifferIndexString {
+	
+	void allDifferIndexOfStringArray(String[] arr1 , String[] arr2){
+		
+		if(arr1.length == arr2.length) {
+			for(int index = 0 ; index < arr1.length ; index++ ) {
+				if(arr1[index] != arr2[index]) {
```

Comment:
For string comparison you should always use .equals() method.

Reply:
Done